### PR TITLE
Adding Style Update

### PR DESCRIPTION
**Pull request by PercyCho**
Adding style update to Petris so it looks cleaner and better for the player.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,7 +93,7 @@
             display: none;
             color: black;
             padding-top: 10px;
-            z-index: 1001;
+            z-index: 11; /*XD Imma change this from 1001 to 11*/
```
Comment (xWafl) [inline index.html]: Don't leave comments unless they contribute to the overall understanding of the code

Also it's at 1001 for a reason; there really should not be anything on top of it.

File: index.html [hunk 2]
```diff
@@ -95,36 +102,43 @@
...
-            border-radius: 2px;
-            -webkit-transition-duration: 0.4s; /* Safari */
+            display: inline;
+            border: #ff9800 solid 3px;
+            border-radius: 1px;
+            -webkit-transition-duration: 0.4s; /* Safari */ 
             -moz-transition-duration: 0.4s; /* Safari */
             transition-duration: 0.4s;
-            /* background-color: rgb(228, 235, 209); */
+            /* background-color: rgb(228, 235, 209); */ /* why a transition...? liek nothing else in the page has transitions, it doesn't really fit the *style* lol */
```
Comment (xWafl) [inline index.html]: I think it looks good and i was planning to add more transitions but I ended up not doing that

File: index.html [hunk 3]
```diff
@@ -145,28 +159,36 @@
...
         #displayusernamediv {
             color: #00E;
-            text-decoration: underline;
+            /*text-decoration: underline;*/
             cursor: pointer;
         }
         #switchdarkdiv {
             color: #00E;
-            text-decoration: underline;
+            /*text-decoration: underline;
```
Comment (xWafl) [inline index.html]: Don't leave comments unless they contribute to the overall understanding of the code

Also I find links pleasing to look at and hover over

File: index.html [hunk 4]
```diff
@@ -246,7 +269,7 @@
             height:100%;
             top:0px;
             left:0px;
-            z-index:1000;
+            z-index: 10; /*XD Imma change this from 1000 to 10*/
```
Comment (xWafl) [inline index.html]: Don't leave comments unless they contribute to the overall understanding of the code

Also it's at 1000 for a reason

File: index.html [hunk 5]
```diff
@@ -277,21 +301,22 @@
...
     </style>
 </head>
- 
 <body scroll="no" id="bodylol">
+    <img id="fadedbackground" src="faded-background-white.png">
     <div id="pauseddiv"><h1>PAUSED</h1></div>
     <div id="usernamediv">
         <div id="maindiv">
-            <h2>Change username</h2>
+            <h1>Change Username</h1>
```
Comment (xWafl) [inline index.html]: don't use more than one <h1> in a site due to google analytics

File: index.html [hunk 6]
```diff
@@ -277,21 +301,22 @@
...
             <form action="/changeusername/submit" method="POST" id="usernameform">
                 <form></form>
-                <span>Username:</span> <input id="formusername" name="formusername" autocomplete="on"></input><br><br>
-                <button class="usernamebutton" id="reset" onclick="resetUsername()">Reset username</button><br><br>
+                <!--we can uuuuuse, Placeholder text!--><input id="formusername" name="formusername" autocomplete="on" placeholder="Username"></input><br><br>
```
Comment (xWafl) [inline index.html]: I considered this but I didn't find it as nice as the current one I thought was better, plus I'm pretty sure placeholder doesn't have screen reader support

File: index.html [hunk 7]
```diff
@@ -310,11 +335,11 @@ <h2>Change username</h2>
     </div>
     <div id="sidebarandselection">
         <div id="sidebar">
-                <a href="/newsb" id="scoreboarda" class="asidebar"><div href="/newsb" id="displayscoreboarddiv" class="sidebardiv">view scoreboard</div></a>
-                <a href="/changelog" id="changeloga" class="asidebar"><div href="/changelog" id="changeloglinkdiv" class="sidebardiv">view changelog</div></a>
-                <div onclick="displayusername()" id="displayusernamediv" class="sidebardiv">change username</div>
-                <div onclick="switchDark()" id="switchdarkdiv" class="sidebardiv">change theme</div>
-                <a href="https://discord.gg/uXks3nd" id="discorda" class="asidebar"><div href="https://discord.gg/uXks3nd" id="discordsidebardiv" class="sidebardiv">Petris discord</div></a>
+                <a href="/newsb" id="scoreboarda" class="asidebar" target="_blank"><div href="/newsb" id="displayscoreboarddiv" class="sidebardiv" style="border-top-width: 6px;">View Scoreboard</div></a>
```
Comment (xWafl) [inline index.html]: Why in a new tab? Current works just fine.

File: index.html [hunk 8]
```diff
@@ -460,14 +485,28 @@ <h1 id="timeleft">You have 10 seconds left to choose piece order.</h1>
...
+            switchDark();
+        }
+        
         function switchDark () {
             if (isDark === false) {
-                document.getElementsByTagName("BODY")[0].style.backgroundColor = "#334";
+                document.getElementsByTagName("BODY")[0].style.backgroundColor = "#1d1d26";
+                document.getElementById('timeleft').style.color = "white";
                 isDark = true;
+                localStorage.setItem('isDark', 'false');
```
Comment (xWafl) [inline index.html]: You can use objects in localStorage, which includes booleans. Not just strings.

**Comment by xWafl** [comment]
Borderline accepted. Was close to rejecting but I can't be bothered to make all the changes myself. The ideas were good, however.

File: index.html (hunk 3 above)
Comment (PercyCho) [reply index.html]: I also find links pleasing to look at! When they're inside of passages of text, that is, not inside of buttons.

**Pull request merged**